tests/loop_test.py

```
@@ -31,10 +31,11 @@ def setUp(self):
         self.loop = None
 
     def _clean_up(self):
-        try:
-            BlockDev.loop_teardown(self.loop)
-        except:
-            pass
+        if self.loop:
+            try:
+                BlockDev.loop_teardown(self.loop)
+            except:
```

Ruff (0.16.3)

[error] Do not use bare `except`

(E722)